st2common/st2common/models/db/rbac.py

```diff
@@ -63,6 +63,7 @@ class UserRoleAssignmentDB(stormbase.StormFoundationDB):
         user: A reference to the user name to which the role is assigned.
         role: A reference to the role name which is assigned to the user.
         description: Optional assigment description.
+        metata: Optional metadata for this assignment.
```

Comment:
spelling of metadata:

Reply:
Thanks, we will actually re-work this a bit.

Spoke to @m4dcoder about it on Slack today and we will actually get rid of metadata, add `source` as a top-level field and add composite index on `user, role, source`.